src/core/manager.h

```diff
@@ -136,7 +136,7 @@ typedef enum ManagerTestRunFlags {
         MANAGER_TEST_FULL = MANAGER_TEST_RUN_BASIC | MANAGER_TEST_RUN_ENV_GENERATORS | MANAGER_TEST_RUN_GENERATORS,
 } ManagerTestRunFlags;
 
-assert_cc((MANAGER_TEST_FULL & UINT8_MAX) == MANAGER_TEST_FULL);
+assert_cc(FLAGS_SET(UINT8_MAX, MANAGER_TEST_FULL));
```

Comment:
hmm, so this check was supposed to check that MANAGER_TEST_FULL fits in 8 bits. The FLAGS_SET() epxression still checks that, but again is a bit misleading, since there are not flags to be checked here...

i tmight make sense to simply rewrite this test as: MANAGER_TEST_FULL <= UINT8_MAX, to make it clearer this is about range